terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverOperations.java

```diff
@@ -67,6 +69,15 @@ public boolean exists(By by) {
         return existsElement;
     }
 
+    /**
+     * Wait until the specified display conditions are met.
+     * @param expectedCondition display condition
+     */
+    public void waitForDisplayed(ExpectedCondition<?> expectedCondition) {
+        WebDriverWait wait = new WebDriverWait(webDriver, defaultTimeoutSecForImplicitlyWait);
+        wait.until(expectedCondition);
+    }
```

Comment:
- [ ] You can choose a better way to provide the `WebDriverWait` field instead of the `waitForDisplayed` method.
`WebDriverOperations` holds only one `WebDriver`, so it is possible to build and hold a `WebDriverWait` for it.

https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/35c9eef561ccab79387798f39e204a3686a335f0/terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverOperations.java#L36

Reply:
a32271ca65b7edb2151d59aa58e448bb7d3d4927

I change to provide `WebDriverWait` field instead of `waitForDisplayed` method .